src/main/java/uk/gov/companieshouse/documentsigningapi/controller/SignDocumentController.java

```diff
@@ -69,6 +73,12 @@ public ResponseEntity<Object> signPdf(final @RequestBody SignPdfRequestDTO signP
         final var key = signPdfRequestDTO.getKey();
         final var map = logger.createLogMap();
         map.put(SIGN_PDF_REQUEST, signPdfRequestDTO);
+
+        List<String> errors = requestValidator.validateRequest(signPdfRequestDTO);
+        if (!errors.isEmpty()) {
+            return buildValidationResponse(BAD_REQUEST.value(), errors, map);
+        }
+
```

Comment:
Did you explore the possibility of using the `@Valid` annotation in this case like we do in some places?